src/components/asset-list/RecyclerAssetList2/FastComponents/FastCurrencySelectionRow.tsx

```diff
@@ -0,0 +1,334 @@
+import React from 'react';
+import { Text as RNText, StyleSheet, View } from 'react-native';
+import {
+  // @ts-ignore
+  IS_TESTING,
+} from 'react-native-dotenv';
+// @ts-ignore
+import { ContextMenuButton } from 'react-native-ios-context-menu';
+import RadialGradient from 'react-native-radial-gradient';
+import { ButtonPressAnimation } from '../../../animations';
+import { CoinRowHeight } from '../../../coin-row';
+import FastCoinIcon from './FastCoinIcon';
+import { Text } from '@rainbow-me/design-system';
+import { useAccountAsset } from '@rainbow-me/hooks';
+import {
+  borders,
+  colors,
+  fonts,
+  fontWithWidth,
+  getFontSize,
+  padding,
+} from '@rainbow-me/styles';
+import { isETH } from '@rainbow-me/utils';
+
+const SafeRadialGradient = (IS_TESTING === 'true'
+  ? View
+  : RadialGradient) as typeof RadialGradient;
+
+export default React.memo(function FastCurrencySelectionRow({
+  item: {
+    uniqueId,
+    showBalance,
+    showFavoriteButton,
+    showAddButton,
+    onPress,
+    theme,
+    nativeCurrency,
+    nativeCurrencySymbol,
+    favorite,
+    toggleFavorite,
+    contextMenuProps,
+    symbol,
+    address,
+    name,
+    testID,
+  },
+}: {
+  item: any;
```

Comment:
Also since we define props [https://github.com/rainbow-me/rainbow/pull/3421/files#diff-1c2b9c8449c98ed1a8daa55c6c57b17426c24c6dd34e7d2461276eb48f006468R236](here) anyway - we should define an interface here.

Reply:
yes, this can be undefined. Zerion is not tracking everything

Reply:
The link is not working

Reply:
we don't have yet typings for this unfortunately. Probably this is currently on @marco's plate.